### ISSUE TITLE
Generated npm script misses -b option

### ISSUE THREAD

**Issue by bodograumann** [chromaui/chromatic-cli#268]
I did my initial chromatic run with

```
npx chromatic --project-token=123456789 -b "storybook:build" -o "storybook-static"
```

given that I have `vue-cli-plugin-storybook` installed.
When asked to generate a 'chromatic' script in package.json I accepted. The script that was written is missing the `-b` option though, i.e. cannot run as-is.

```
"chromatic": "npx chromatic storybook:build -o storybook-static --project-token 123456789"
```

**One more thing**:
In this case I have `chromatic` installed as a dependency, so the `npx` can be left away in the script.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1199921881886840/1200139909630251) by [Unito](https://www.unito.io)

**Comment by ghengeveld** [comment]
I tried to reproduce this, but it works for me.

<img width="1515" alt="Screenshot 2021-05-07 at 15 09 12" src="https://user-images.githubusercontent.com/321738/117454367-5abf7380-af46-11eb-9935-9b10e71279e9.png">

It still prefixes the script with `npx` because it doesn't check whether you actually installed `chromatic` as a local dependency. Using `npx` works either way and will use a locally installed dependency if there is one.

**Comment by ghengeveld** [comment]
Confirmed and fixed in #337
